test/integration/decorator/decorator_test.go

```diff
@@ -18,6 +18,8 @@ package decorator
 
 import (
 	"context"
+	v12 "metacontroller/pkg/controller/common/customize/api/v1"
```

Comment:
should the name `v12` be something like `customizev1` ?

Reply:
oh, right, this was automatically added by intellij, will change it